### PR TITLE
Features/headers and reactions

### PR DESCRIPTION
**Pull request by minybrandon**
We created two components for the header: publishedDate and userId.

### CODE REVIEW

File: src/post/userId.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class UserId extends React.Component{
+    render(){
+        return(
+            <p>{this.props.userId}</p>
+        )
+    }
+}
```
Comment (opmanual) [inline src/post/userId.js]: Need to add the code below for using other components
"export default UserId;"

File: src/post/publishedDate.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'; 
+
+class PublishedDate extends React.Component{
+    render(){
+        return(
+            <p>{this.props.publishedDate}</p>
+        )
+    }
+}
```
Comment (opmanual) [inline src/post/publishedDate.js]: Need to add the code below for using other components
"export default PublishedDate;"

**Review by opmanual — CHANGES_REQUESTED** [review]

File: src/post/publishedDate.js (hunk 2 above)
Comment (sangyum) [reply src/post/publishedDate.js]: Fix plz 😢

**Review by sangyum — CHANGES_REQUESTED** [review]
you need to export components

File: src/post/userId.js (hunk 1 above)
Comment (sangyum) [reply src/post/userId.js]: What a big mistake! 👎

File: src/post/header.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PublishedDate from './publishedDate'
+import UserId from './userId'
+
+class Header extends React.Component{
+
+    render() {
+        return( 
+            <>
+                <PublishedDate/>
```
Comment (sangyum) [inline src/post/header.js:10]: This could have been implemented as a single component.

File: src/post/header.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+                <UserId/>
+            </>
+        )
+    }
+}
+
+export default Header
```
Comment (sangyum) [inline src/post/header.js]: Add a new line after export

File: src/post/publishedDate.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+export default PublishedDate
```
Comment (sangyum) [inline src/post/publishedDate.js]: add a new line after export

File: src/post/userId.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+export default UserId
```
Comment (sangyum) [inline src/post/userId.js]: add a new line after export

**Review by sangyum — CHANGES_REQUESTED** [review]
more changes please

**Review by sangyum — APPROVED** [review]
Have a 🌮

File: src/post/header.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PublishedDate from './publishedDate'
+import UserId from './userId'
```
Comment (justinyum98) [inline src/post/header.js]: Rename the `publishedDate.js` and `userId.js` files to `PublishedDate.js` and `UserID.js`
Change `./publishedDate` and `./userId` to `./PublishedDate` and `./UserID`

React components should follow Pascal case, not camel case.
[Read this.](https://www.quora.com/What-is-the-difference-between-Pascal-Case-and-Camel-Case-in-computer-programming#:~:targetText=Pascal%20case%20is%20a%20subset,UserAccount%20is%20a%20Pascal%20case.&targetText=Pascal%20is%20a%20proper%20noun,not%20capitalize%20the%20first%20letter.)

File: src/post/header.js (hunk 3 above)
Comment (justinyum98) [reply src/post/header.js:10]: Also, add a space between `PublishedDate` and `/>`

File: src/post/header.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+class Header extends React.Component{
+
+    render() {
+        return( 
+            <>
+                <PublishedDate/>
+                <UserId/>
```
Comment (justinyum98) [inline src/post/header.js:11]: Add a space between `UserId` and `/>`

**Review by tjdalsans — APPROVED** [review]

**Review by justinyum98 — CHANGES_REQUESTED** [review]
Make changes 👎

**Review by justinyum98 — APPROVED** [review]
👍

**Review by tjdalsans — APPROVED** [review]

**Review by opmanual — APPROVED** [review]

**Pull request merged**